examples/multiple-choice/index.es

```diff
@@ -0,0 +1,77 @@
+Element `multiple-choice`
```

Comment:
@dees I mean the library piece that supports the multiple-choice. Can't find it anywhere.